src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -119,4 +119,79 @@
             @include text-header6($font-weight-regular);
         }
     }
+
+    @media (min-width: $zero-width) and (max-width: $tablet-width) {
+        .documentBox {
+            @include flex-position(row, nowrap, space-between, center);
+            width: 100%;
+            border: $border-size solid $gray-300;
+            border-radius: $border-radius--normal;
+            padding: 0.5rem;
+            background-color: $white;
+            gap: 2.5rem;
+            &:deep(svg) {
+                color: $blue-700;
+                height: 2rem;
+            }
+            &__title {
+                @include text-header6--tablets($font-weight-medium);
+                color: $gray-900;
+                text-align: center;
+            }
+            &__data {
+                @include text-header6--tablets($font-weight-medium);
+                text-align: center;
+            }
+        }
+    }
+
+    @media (min-width: $tablet-above-width) and (max-width: $laptop-width) {
```

Comment:
why var above it? I think the better idea is to set min-width for laptop width and delete max width . Then next grade of media will be starting from min width e.g 1024px

Reply:
zrobilam to by zgodnosc byla wszedzie, oznacza to ze jak dodalam z tym zerem to chcialam to wszedzie dodac by bylo jednolicie.